docs/user_guide.md

```diff
@@ -64,61 +67,19 @@ pip install shark-ai[apps]
 python -m shortfin_apps.sd.server --help
 ```
 
-## Quickstart
+## Getting started
 
-### Run the SDXL Server
+As part of our current release we support serving [SDXL](https://stablediffusionxl.com/) and [Llama 3.1](https://ai.meta.com/blog/meta-llama-3-1/) variants as well as an initial release of <code>sharktank</code>, SHARK's model development toolkit which is leveraged in order to compile these models to be high performant.
```

Comment:
Prefer markdown over HTML in `.md` files. Most markdown renderers support HTML, but some limit the HTML tags that they support. The `<code>` tag is probably fine, but backticks are simpler :P

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

```suggestion
As part of our current release we support serving [SDXL](https://stablediffusionxl.com/) and [Llama 3.1](https://ai.meta.com/blog/meta-llama-3-1/) variants as well as an initial release of `sharktank`, SHARK's model development toolkit which is leveraged in order to compile these models to be high performant.
```